cli.js

```diff
@@ -129,7 +126,6 @@ if (
 var api = new Api({
 	failFast: cli.flags.failFast,
 	serial: cli.flags.serial,
-	require: arrify(cli.flags.require),
```

Comment:
Before, `cli.flags.require` would default to the config from `package.json`, through [this line](https://github.com/jacobkahn/ava/blob/f94b58df23202749691e3e82305486efd6a9854d/cli.js#L98). `Api` still needs the `require` option, so this should now be `require: arrify(conf.require),`.